sources/assets/zsh/zshrc

```diff
@@ -84,3 +84,15 @@ else
   [ -d /opt/my-resources/setup/zsh ] || mkdir -p /opt/my-resources/setup/zsh
   cp /.exegol/skel/zsh/zshrc /opt/my-resources/setup/zsh/zshrc
 fi
+
+fzf-haiti () {
+    haiti -e "$1" | fzf --ansi | grep -E -o ' [0-9]*]' | cut -d ']' -f 1 | cut -d ' ' -f 2
+}
+
+hashcat2 () {
+    hash="$1"
+    shift 1
+    hashcat -m `fzf-haiti "$hash"` "$hash" `fzf-wordlists` "$@"
+}
+
+alias hcat='hashcat2'
```

Comment:
I think it would be possible to do something like `alias hcat='hashcat -m $(fzf-haiti "$1") "$1" $(fzf-wordlists) "${@:2}"'`. In which case it should be added in `sources/assets/zsh/aliases.d/fzf` as well